store/sqlstore/store.go

```diff
@@ -1627,3 +1627,11 @@ func (ss *SqlStore) jsonDataType() string {
 	}
 	return "json"
 }
+
+func (ss *SqlStore) toReserveCase(str string) string {
+	if ss.DriverName() == model.DatabaseDriverPostgres {
+		return fmt.Sprintf(`"%s"`, str)
```

Comment:
We can use the `%q` verb here. `fmt.Sprintf("%q", str)`

Reply:
@agnivade PR updated